src/components/Help/Steppers/Stepper.scss

```diff
@@ -0,0 +1,11 @@
+.box {
+    width: '100%';
+    height: 600px;
+    border: 5px solid black;

```

Comment:
Can you make this `4px` instead? I think web dev usually prefers even numbers.

Reply:
Corrected